pkg/client/results/processing.go

```diff
@@ -146,48 +148,120 @@ func PostProcessPlugin(p plugin.Interface, dir string) (Item, error) {
 	return i, err
 }
 
+// processNodesWithProcessor is called to invoke processDir on each node-specific directory contained
+// underneath the given dir. The directory is assumed to be either the results directory or errors directory
+// which should have the nodes as subdirectories. It returns an item for each node processed and an error
+// only if it couldn't open the original directory. Any errors while processing a specific node are logged
+// but not returned.
+func processNodesWithProcessor(p plugin.Interface, baseDir, dir string, processor postProcessor, selector fileSelector) ([]Item, error) {
```

Comment:
This method is just code moved code, not new code. Extracted from the `processPluginWithProcessor` method.